api/external_resources/signals.py

```diff
@@ -0,0 +1,93 @@
+import logging
+
+from django.db.models.signals import post_save, pre_delete
+from django.dispatch import receiver
+
+from features.models import FeatureState
+from integrations.github.tasks import call_github_app_webhook_for_feature_state
+from webhooks.webhooks import WebhookEventType
+
+# noinspection PyUnresolvedReferences
+from .models import ExternalResources
+
+logger = logging.getLogger(__name__)
+
+
+@receiver(post_save, sender=ExternalResources)
```

Comment:
can we use a hook for this instead of a signal?

Reply:
Yes, it is possible. I converted them into hooks. Anyway, I tried to base it on the original webhook that used signals, which was triggered when there was a change in the feature state.